### ISSUE TITLE
[Suggestion] Balance number of AI Titans according to the number of players.

### ISSUE THREAD

**Issue by BTFighter** [DBmaoha/Super.Mixed.Game#34]
As said in the title.

**Comment by DBmaoha** [comment]
I'm preparing an update that makes npc spawn count modifiable through Playlist Variables, adding this feature could be much too complicated. Maybe you can try set titan count to a specific number depending on your server's max players limit when the new update is released?

**Comment by BTFighter** [comment]
That's fine, modifying the mod directly is what I've been doing anyways. I'll close the issue once the update is posted.

**Comment by BTFighter** [comment]
Forgot to ask, does this allow to set the number of titans in a team?
Ex. 4 allied titans and 5 enemy titans.

**Comment by DBmaoha** [comment]
Tried to add playlistvar settings in [v1.7.0](), might be a bit complicated... I'm trying to cover most stuffs that can be changed

Sorry it takes very long time to fully implement. I was busy recently and when I have time I'm mostly focusing on adding new contents or doing some debug. Now most of my toughts have been completed and I think mod's codebase is stable enough for adding more settings. All available settings are listed in thunderstore page.

For diffrent team npc count, it's not added because it's kinda complicated and balancing may seem bad. But in case you want to play it with only yourself or staying at the same team with your friends, I've added `aitdm_extended_coop` and related settings that allows you and your friends to always stay at the same team against enemy npcs, no pvp.

**Comment by DBmaoha** [comment]
Added playlistvar settings in [v1.7.0](https://northstar.thunderstore.io/package/VoyageDB_Modding_Home/Attrition_Extended/), might be a bit complicated... I'm trying to cover more stuffs that can be changed

Sorry it takes very long time to fully implement. I was busy recently and when I have time I'm mostly focusing on adding new contents or doing some debug. Now most of my toughts have been completed and I think mod's codebase is stable enough for adding more settings. All available settings are listed in thunderstore page.

For diffrent team npc count, it's not added because it's kinda complicated and balancing may seem bad. But in case you want to play it with only yourself or staying at the same team with your friends, I've added `aitdm_extended_coop` and related settings that allows you and your friends to always stay at the same team against enemy npcs, no pvp.
